src/scss/tooltip.scss

```diff
@@ -4,7 +4,7 @@
 /// Base tooltip style. Inherits from oOverlay.
 @mixin oTooltip {
 	@include oOverlay();
-	@include oEffectsShadowsElevation($elevation: 'high');
+	@include oVisualEffectsShadowsElevation($elevation: 'high');
 
 	// Override border until it's updated in o-overlay
```

Comment:
I think you can probably remove this as it should have been updated in o-overlay now.